src/main/java/mate/academy/onlinebookstore01/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,24 @@
 package mate.academy.onlinebookstore01.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @Min(1)
     private Long id;
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
+    @Size(min = 10, max = 13)

```

Comment:
ISBN has fixed format, you can use [Hibernate validators](https://www.baeldung.com/hibernate-validator-constraints) or rewrite your regexp by [this](https://howtodoinjava.com/java/regex/java-regex-validate-international-standard-book-number-isbns/) guide